docs/source/user/file_formats.md

```diff
@@ -1,162 +1,151 @@
 
-## HTML
-
-* File format: edit as text file
-* MIME type: `text/html`
-
-JupyterLab supports rendered HTML in output. HTML files can be edited as text
-files in the File Editor.
+[animation]
 
-## SVG
+## CSV
 
-* File format: `.svg`
-* MIME type: `image/svg+xml`
+* File extension: `.csv`
+* MIME type: None
 
-JupyterLab will render Scalable Vector Graphics (SVG) in files and output. SVG
-files can slso be edited as text files in the File Editor.
+Files with rows of comma-separated values (CSV files) are a common
+format for tabular data. The default viewer for CSV files in JupyterLab is a high-performance data grid viewer:
 
-## LaTeX
+[animation]
 
-* File format: edit as text file
-* MIME type: `text/latex`
+To edit a CSV file as a text file, right-click on the file in the file browser
+and select the “Editor” item in the “Open With…” submenu:
 
-JupyterLab will render LaTeX questions in output, and LaTeX files (`.tex`) can
-be edited as text files in the File Editor.
+[animation]
 
 ## JSON
 
-* File format: `.json`
-* MIME type: `application+json`
+* File extension: `.json`
+* MIME type: `application/json`
 
-JavaScript Object Notation (JSON) files are common in data science.
+JavaScript Object Notation (JSON) files are common in data science. JupyterLab supports displaying JSON data in cell output or viewing a JSON file using a searchable tree view:
 
-The default viewer for JSON files is a searchable tree view:
+[animation showing both rendering JSON as cell output and viewing a JSON file]
 
-[animation]
-
-To edit the JSON as a text file, right-click on the file in the File Browser and
-select the “Editor” item in the “Open With…” submenu:
+To edit the JSON as a text file, right-click on the filename in the file browser
+and select the “Editor” item in the “Open With…” submenu:
 
 [animation]
 
-## CSV
+## HTML
 
-* File format: `.csv`
-* MIME type: None
+* File extension: `.html`
+* MIME type: `text/html`
 
-Files with rows of Comma Separate Values (with a `.csv` extension) are a common
-format for tabular data.
+JupyterLab supports rendering HTML in cell output and editing HTML files as text in the file editor.
 
-The default viewer for CSV files in JupyterLab is a high performance data grid
-viewer. To view a CSV file in the interactive Data Grid, double-click on the
-file in the File Browser:
+## LaTeX
 
-[animation]
+* File extension: `.tex`
+* MIME type: `text/latex`
 
-To edit a CSV file as a text file, right-click on the file in the File Browser
-and select the “Editor” item in the “Open With…” submenu:
+JupyterLab supports rendering LaTeX in cell output and editing LaTeX files as text in the file editor.
```

Comment:
We don't support full LaTeX in cell outputs, just LaTeX math mode.

Reply:
I waffled about the description here. You still have to put the latex in delimiters, though - so that might be confusing.

Reply:
Yeah, LaTeX support is similar to Markdown, in that it is not always obvious/spec'ed out.